pages/brand.js

```diff
@@ -237,8 +237,9 @@ const Page = ({ css }) => (
       </Box>
       <Heading variant="headline">Icons</Heading>
       <Text variant="subtitle" sx={{ a: { variant: 'styles.a' }, mb: 3 }}>
-        We have a custom iconset, published as{' '}
-        <a href="https://github.com/hackclub/icons">@hackclub/icons</a>.
+        <p>We have a custom iconset, published as{' '}
+          <a href="https://github.com/hackclub/icons">@hackclub/icons</a>.
+        </p>
```

Comment:
```suggestion
        We have a custom iconset, published as{' '}
          <A href="https://github.com/hackclub/icons">@hackclub/icons</A>.
```
Instead of using `p` tags, you could add a `as="p"` to the `Text` component. 

Using the theme-ui `A` (link) component also might make it cleaner so you don't need to add `a: { variant: 'styles.a' }`

Reply:
Smart!! I haven't used theme-ui before so thank you for guiding me 🙇